core/core-api/src/main/java/io/fabric8/launcher/core/api/events/StatusEventType.java

```diff
@@ -10,6 +10,7 @@
     GITHUB_PUSHED("Pushing your customized Booster code into the repo"),
     OPENSHIFT_CREATE("Creating your project on OpenShift"),
     OPENSHIFT_PIPELINE("Setting up your build pipeline"),
+    OSIO_CODEBASE_CREATED("Setting OSIO Codebase"),
```

Comment:
@gastaldi I am not happy about mixing OSIO and upstream through events. Do you have any suggestion on how to decouple it?

Reply:
Extract the enum into an interface and have separate enums for each mission control implementation?